From: NeilBrown <neilb@suse.de>
Subject: Fix dm-raid45 to match previous patch
Patch-mainline: not-relevant
Reference: FATE#313903

.status now takes a 'status_flags' argument, which says whether to
flush before reporting status or not.  Not relevant to raid45.

Signed-off-by: NeilBrown <neilb@suse.de>
Acked-by: NeilBrown <nfbrown@suse.com>

---
 drivers/md/dm-raid45.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- linux-3.0-SLE11-SP3.orig/drivers/md/dm-raid45.c
+++ linux-3.0-SLE11-SP3/drivers/md/dm-raid45.c
@@ -4341,7 +4341,7 @@ static void raid_devel_stats(struct dm_t
 }
 
 static int raid_status(struct dm_target *ti, status_type_t type,
-		       char *result, unsigned maxlen)
+		       unsigned status_flags, char *result, unsigned maxlen)
 {
 	unsigned p, sz = 0;
 	char buf[BDEVNAME_SIZE];
